sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_28_days_later_v1/query.sql

```diff
@@ -0,0 +1,77 @@
+WITH clients_first_seen_28_days_ago AS (
+  SELECT
+    client_id,
+    sample_id,
+    country AS country_code,
+    normalized_channel AS channel,
+    app_build_id AS build_id,
+    normalized_os AS os,
+    mozfun.norm.truncate_version(normalized_os_version, "minor") AS os_version,
+    attribution_source,
+    distribution_id,
+    attribution_ua,
+    first_seen_date,
+  FROM
+    telemetry_derived.clients_first_seen_v2
+  WHERE
+    first_seen_date = DATE_SUB(@submission_date, INTERVAL 27 day)
+    -- TODO we will need to wait till startup_profile_selection_reason is backfilled to clients_daily before uncommenting the following line
+    -- AND startup_profile_selection_reason = 'firstrun-created-default'
```

Comment:
I think we may want to move `startup_profile_selection_reason` to the select statement, since *I think* this table is meant to show retention outcomes for ALL new profiles.